corehq/apps/callcenter/utils.py

```diff
@@ -23,7 +24,15 @@
 from django.core.cache import cache
 
 
-class DomainLite(namedtuple('DomainLite', 'name default_timezone cc_case_type use_fixtures')):
+@attr.s
+class DomainLite(object):
+    name = attr.ib()
+    default_timezone = attr.ib()
+    cc_case_type = attr.ib()
+    use_fixtures = attr.ib()
+    form_datasource_enabled = attr.ib()
+    case_datasource_enabled = attr.ib()
+    case_actions_datasource_enabled = attr.ib()
```

Comment:
suggestion: set defaults so only non-default values are required for construction.